zendesk_attachment_backup.py

```diff
@@ -69,37 +73,51 @@ def collect_urls(input_files, print_urls, **kwargs):
 
    # Print URLs to file if requested
    if print_urls:
-      with open('attachment_urls.txt', "w") as fh:
+      target_file=os.path.join(kwargs.get("output_dir"), 'attachment_urls.txt')
```

Comment:
No, you _should_ use spaces around assignments all around everywhere. :wink: Like:
```python
# Wrong
a=42

# Right
a = 42
```